package.json

```diff
@@ -101,14 +101,13 @@
       "<rootDir>/src.*",
       "<rootDir>/node_modules.*",
       "<rootDir>/app/src.*",
-      "<rootDir>/app/dist.*",
```

Comment:
Good catch. I had a valuable intention, though: to exclude the webpack output folder (which is just packing of tsc's output, useless to test/watch, so let's not test/watch). So, pls keep the line but replace `dist` with `lib`.